### PR TITLE
STORM-1055

### PR DESCRIPTION
**Pull request by randerzander**
Fixed README by adding note about com.google.common.collect class usage, also fixed hikariConfigMap reference in example.

### CODE REVIEW

**Comment by randerzander** [comment]
Not sure why the CI build failed, as I only modified the storm-jdbc README..

File: external/storm-jdbc/README.md [hunk 1]
```diff
@@ -3,14 +3,16 @@ Storm/Trident integration for JDBC. This package includes the core bolts and tri
...
 in a storm topology.
 
+**Note**: Throughout the examples below, we make use of com.google.common.collect.Lists and com.google.common.collect.Maps.
+
 ## Inserting into a database.
 The bolt and trident state included in this package for inserting data into a database tables are tied to a single table.
 
 ### ConnectionProvider
-An interface that should be implemented by different connection pooling mechanism `org.apache.storm.jdbc.common.ConnectionProvider`
+An interface that should be implemented by different connection pooling mechanism `org.apache.storm.jdbc.common.ConnectionPrvoider`
```
Comment (HeartSaVioR) [inline external/storm-jdbc/README.md]: ConnectionProvider is correct, so it shouldn't be changed.

File: external/storm-jdbc/README.md [hunk 2]
```diff
@@ -3,14 +3,16 @@ Storm/Trident integration for JDBC. This package includes the core bolts and tri
...
 
 ```java
-public interface ConnectionProvider extends Serializable {
+public interface ConnectionPrvoider extends Serializable {
```
Comment (HeartSaVioR) [inline external/storm-jdbc/README.md]: Again, ConnectionProvider is correct, so it shouldn't be changed.

**Comment by HeartSaVioR** [comment]
You don't need to consider about Travis CI since you modifies only markdown file.
It's due to random test failures.

File: external/storm-jdbc/README.md (hunk 1 above)
Comment (randerzander) [inline external/storm-jdbc/README.md]: I cloned from master, not sure where this change came from. Corrected in the next commit.

**Comment by HeartSaVioR** [comment]
+1.
Since it modifies only doc., I'll merge into master right now.

**Comment by randerzander** [comment]
Unfortunately I missed an additional map->hikariConfigMap example further down the README. Also, this applies to the 0.10.x branch as well. Is it possible to merge there too?

**Comment by HeartSaVioR** [comment]
@randerzander 
Please request another PR with same title. Please leave a reminder to new PR so that I can take care of merging into 0.10.x-branch at that time.

**Pull request merged**